cmd/clusterctl/client/cluster/mover.go

```diff
@@ -443,7 +443,7 @@ func (o *objectMover) restore(graph *objectGraph, toProxy Proxy) error {
 
 	// Resume reconciling the ClusterClasses after being restored from a backup.
 	// By default, during backup, ClusterClasses are paused so they must be unpaused to be used again
-	log.V(1).Info("Resuming the target cluter classes")
+	log.V(1).Info("Resuming the target cluster classes")
```

Comment:
nit
```suggestion
	log.V(1).Info("Resuming the target ClusterClasses")
```

And the same elsewhere 🙂